template/manifests/PodDisruptionBudget/manifest/draft.yaml

```diff
@@ -0,0 +1,42 @@
+templateName: "podDisruptionBudget-manifest"
+description: "This template is used to create a PodDisruptionBudget for an application"
+type: "podDisruptionBudget"
+variables:
+  - name: "APPNAME"
+    type: "string"
+    kind: "resourceName"
+    description: "the name of the application"
+  - name: "PARTOF"
+    type: "string"
+    kind: "part-of"
```

Comment:
nit: can you make this camelCase to maintain uniformity

Reply:
it is part of the label: app.kubernetes.io/part-of: online-store, so I kept it as is!

Reply:
Makes sense